### DISCUSSION TITLE
Can I reference environment variables within each other?

### DISCUSSION THREAD

**Discussion by rohan** [vercel/community#505] [Help] [answered]
(I have a workaround for this, but I'm just curious...)

Auth0 has a clean setup version that reads environment variables. One of them is the base URL, which in my case is the app's URL. I know I can read this URL as `process.env.VERCEL_URL`, but I'm curious: is there a way to use the environment variables editor to set `AUTH0_BASE_URL` to `VERCEL_URL` (or, in shell, `AUTH0_BASE_URL=$VERCEL_URL`)?

**Comment by samcx** [comment, marked as answer]
Unfortunately, this is not possible to do from our UI. You will need to do this programatically.

Let us know if you have any further questions or concerns!

**Reply by jonworek** [reply]
@samcx could you provide an example?

**Comment by BrimmedHat** [comment]
I had the same question. Out of curiosity could you share your workaround?
